src/pages/monthly-plan.js

```diff
@@ -62,6 +52,24 @@ const suggestedAmounts = [
   },
 ];
 
+const getProfileData = (id, accessToken, excludedRepos) => {
+  const params = { id };
+  if (excludedRepos) {
+    params.excludedRepos = excludedRepos;
+  }
+  const searchParams = new URLSearchParams(params);
+  return process.env.IS_CLIENT
+    ? fetchJson(`/data/getProfileData?${searchParams}`)
+    : import('../lib/data').then(m =>
+        m.getProfileData(id, accessToken, { excludedRepos }),
+      );
+};
+
+const getFilesData = sessionFiles =>
```

Comment:
Why do we have that there and not in `fetch.js` ?

Reply:
I initially moved it to fetch.js  as an abstraction because it was  both in profile and monthly plan and created a single function `getData` to use for both file and profile type. But you asked me to remove it and revert back. I forgot to delete it from fetch.js after removing `getData`.